### DISCUSSION TITLE
Installing git-lfs on Bonobo git server

### DISCUSSION THREAD

**Discussion by vitalik198** [git-lfs/git-lfs#5471] [Q&A]
Hi!

When I try to push changes to the server, I get the error.

```
batch response: Repository or object not found: http://123/BonoboGitServer/somerepo.git/info/lfs/objects/batch
Check that it exists and that you have proper access to it
```

At the same time, the address http://123/BonoboGitServer/somerepo.git is correct and opens successfully in the browser, but the routing that follows is not working.

What exactly is required for the lfs extension to work?
Can I configure BonoboGitServer to support git-lfs?

**Comment by chrisd8088** [comment]
Hi, thanks for the question!  I don't think any of the Git LFS project's core team have any experience with the Bonobo Git Server, but perhaps other people do and can chime in.

I would also suggest reaching out to the Bonobo project itself; it looks like they ~have a Google Groups [forum](https://groups.google.com/g/bonobo-git-server) for questions, for instance~ accept questions as issues in their GitHub `jakubgarfield/Bonobo-Git-Server` [repository](https://github.com/jakubgarfield/Bonobo-Git-Server/issues).  The jakubgarfield/Bonobo-Git-Server#342 issue looks particularly relevant, and suggests there is no Git LFS support in the Bonobo Git Server yet.  Please do let us know what you find out!

**Comment by vitalik198** [comment]
It looks like the simplest solution would be to use a different server implementation.
I already deployed gogs and the problem was solved.
